shared/chat/conversation/list-hoc.js

```diff
@@ -7,10 +7,10 @@ import {withProps} from 'recompose'
 import type {Props} from './list'
 import type {Options} from './messages'
 
-type OptionsFn = (message: ChatConstants.Message, prevMessage: ChatConstants.Message, isFirstMessage: boolean, isSelected: boolean, isScrolling: boolean, key: any, style: Object, onAction: () => void, isEditing: boolean) => Options
+type OptionsFn = (message: ChatConstants.Message, prevMessage: ChatConstants.Message, isFirstMessage: boolean, isSelected: boolean, isScrolling: boolean, key: any, style: Object, onAction: () => void, onShowEditor: (event: any) => void, isEditing: boolean) => Options
 
 function propsToMessageOptionsFn (props: Props): OptionsFn {
-  return function (message, prevMessage, isFirstMessage, isSelected, isScrolling, key, style, onAction, isEditing = false): Options {
+  return function (message, prevMessage, isFirstMessage, isSelected, isScrolling, key, style, onAction, onShowEditor, isEditing = false): Options {
```

Comment:
not ok, https://github.com/keybase/client/blob/6aae51889319e14a3f2c3a1d2c328e64704686f3/shared/chat/conversation/list.native.js#L134

It'll pass the wrong type.

while you're at it, could you update https://github.com/keybase/client/blob/6aae51889319e14a3f2c3a1d2c328e64704686f3/shared/chat/conversation/list.js.flow `optionsFn: any` to `optionsFn: OptionsFn` and move OptionsFn to list.js.flow, and reference that type here.